src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1c14bb4d51.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To Do List</title>
+  </head>
+  <body>
+    <section class="to-do-container">
+      <div class="to-do-inner">
+        <div class="title">
+          <h3>Todays's To Do</h3>
+          <i class="fa-solid fa-arrows-rotate"></i>
+        </div>
+        <input class="add-btn" type="text" placeholder="Add to your list..." />
+        <div id="list-container"></div>
+      </div>
+      <div class="btn-container">
+        <button class="clear-btn">Clear all completed</button>
+      </div>
+    </section>
```

Comment:
- [ Required ] I think it would be nice if the styling of the to-do list in your application matches the design in the video provided for this milestone. Currently, it appears that your design varies from the one provided in the project requirements in the following ways.
   
    - [ ] The clear all todos `button` should occupy the entire width of the todo list and have a greyish background color as shown in the project video. 
    - [ ] The todos on the todo list should have a `border-bottom` styling, currently, your design for the todo list items has no borders specified.

Please see the screenshots below and rectify this.

## Project design
![todolist-project](https://github.com/hamzaw9/ToDoList/assets/16392046/4982b64f-e26a-4be9-8eeb-c8fc87b35a41)

## Your design
![todolist-student](https://github.com/hamzaw9/ToDoList/assets/16392046/0c0367c9-7843-4e55-be02-ed8151816b9b)